### PR TITLE
Add username

### LINKED ISSUE #1270: Sadeedpv
## Profile Screenshot

github

### PR DESCRIPTION
**Pull request by Sadeedpv**
resolves #1270

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Sadeedpv.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/96517901/181920820-ecc4d039-6a4a-436c-83d5-a1d12707ce79.png",
+  "issueId": 1270,
+  "name": "Sadeed pv",
+  "username": "Sadeedpv"
+}
```
Comment (Panquesito7) [inline profiles/Sadeedpv.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Sadeedpv.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (Panquesito7) [inline profiles/Sadeedpv.json:7]: There's an extra line here. GitHub doesn't display the endline.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**